steps/datadog-mobile-app-run-tests/1.0.0/step.yml

```diff
@@ -0,0 +1,113 @@
+title: datadog-mobile-app-run-tests
+summary: |
+  Step to run Datadog Synthetics tests on your CI
+description: |
+  With the `synthetics-test-automation-bitrise-step-run-tests` step you can run Synthetics tests during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle.
+website: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests
+source_code_url: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests
+support_url: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/issues
+published_at: 2024-03-01T10:58:14.103768+01:00
+source:
+  git: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git
+  commit: 2239150885d7cce9cec0abdd4072cd51b01ecdc0
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: run-tests.sh
+inputs:
+- api_key: $DATADOG_API_KEY
+  opts:
+    description: The API key used to query the Datadog API.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: Datadog API Key
+- app_key: $DATADOG_APP_KEY
+  opts:
+    description: The application key used to query the Datadog API.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: Datadog APP Key
+- config_path: ""
+  opts:
+    description: The global JSON configuration used when launching tests.
+    title: Global JSON configuration
+- device_ids: ""
```

Comment:
Note: you could use the `category:` property of the inputs to display them in the same expandable block in the workflow editor. (See an example here: https://github.com/bitrise-steplib/steps-xcode-test/blob/4e14f19dd7ad5a9bcb1318bcbdab3b8d9ef63ce3/step.yml#L116) This would make Step easier to configure.

The test if it works correctly you can use `bitrise :workflow-editor` command locally in the step repository (when referencing it using _path::_).

Reply:
Thank you, it does look nicer this way 👍 
Split the inputs in two categories - required and optional